docs/terms-policies/governance.md

```diff
@@ -0,0 +1,267 @@
+5. Stewardship: Authority derives from consistent, high‑quality contribution.
+6. Accountability: Roles carry explicit responsibilities.
+7. Security & Privacy: Responsible handling of sensitive data and credentials.
+
+## 4. Project Structure
+
+The DANDI Project codebase includes the components listed in the table below:
+
+| Domain | Primary Repos |
+|--------|------------------------------|
+| Archive | [dandi-archive](https://github.com/dandi/dandi-archive), [dandi-infrastructure](https://github.com/dandi/dandi-infrastructure) |
+| Client | [dandi-cli](https://github.com/dandi/dandi-cli), [dandidav](https://github.com/dandi/dandidav) |
+| Metadata | [dandi-schema](https://github.com/dandi/dandi-schema), [schema](https://github.com/dandi/schema) |
+| JupyterHub | [dandi-hub](https://github.com/dandi/dandi-hub), [nebari](https://github.com/dandi/nebari), [nebari-deployments](https://github.com/dandi/nebari-deployments), [nebari-docker-images](https://github.com/dandi/nebari-docker-images) |
+| Documentation & Support | [dandi-docs](https://github.com/dandi/dandi-docs), [dandi-about](https://github.com/dandi/dandi-about), [helpdesk](https://github.com/dandi/helpdesk) |
+
+Multiple deployments of this codebase operate as separate instances, each serving distinct research communities and datasets. The DANDI Archive instance is available at https://dandiarchive.org/, and the EMBER-DANDI Archive instance is available at https://dandi.emberarchive.org/. This document governs over the DANDI Project codebase unless otherwise specified.
+
+Contributing code to DANDI repositories does not grant rights or access to the separate instances. Similarly, using the DANDI Archive or EMBER-DANDI Archive service does not require contributing to the codebase.
+
+## 5. Roles & Responsibilities
+
+### 5.1 Contributors
+Anyone submitting issues, pull requests, documentation, or feedback.
+
+Responsibilities:
+- Follow [Code of Conduct](https://github.com/dandi/dandi-archive/blob/master/CODE_OF_CONDUCT.md) and contribution guidelines.
+- Strive to provide sufficient context and steps to reproduce.
+- Where applicable, write tests and documentation for code changes.
+
+### 5.2 Reviewers
+Contributors granted reviewer status for designated repositories.  
+
+Responsibilities:
+- Perform timely, constructive reviews.
+- Enforce style, testing, and security practices.
+- Identify architectural and performance impacts.
+
+Path to role:
+- Consistent high‑quality reviews.
+- Sponsored by at least one Maintainer.
+
+### 5.3 Maintainers
+Individuals with merge rights for designated repositories.  
+
+Responsibilities:
+- Merge approval.
+- Release planning and tagging.
+- Triage (labels, prioritization, assignment).
+- Manage vulnerability reports.
+- Escalate policy or security concerns.
+- Facilitate cross‑repository alignment.
+- Onboard and mentor reviewers.
+
+Expectations:
+- Active presence.
+- Adhere to conflict of interest and bias avoidance.
```

Comment:
This should link to the conflict of interest section (6.2 below).

Is "bias" defined anywhere?